public/app/plugins/panel/geomap/layers/data/markersLayer.tsx

```diff
@@ -99,6 +100,11 @@ export const markersLayer: MapLayerRegistryItem<MarkersConfig> = {
       legend = <ObservablePropsWrapper watch={legendProps} initialSubProps={{}} child={MarkersLegend} />;
     }
 
+    const markerPath =
+      getPublicOrAbsoluteUrl(config.markerSymbol?.fixed) ?? getPublicOrAbsoluteUrl('img/icons/marker/circle.svg');
+
+    const uri = await getSVGUri(markerPath, config.size.fixed * 2);
```

Comment:
so size here is based on the data... how does that relate to the 24 hardcoded in the svg proper?

Does it make sense to load the svg at a fixed size (24? 100?) and then scale it externally?

Reply:
The size should set the height and width attributes of the svg. These numbers were an initial pass to bring the svg sizing and scaling behavior closer to how the regular shapes are sized and scaled, while also allowing for the tooltip to display for svgs.

Ideally, it would be great to not have to set a fixed height and width for the svg - but we were encountering an issue with open layers not displaying the tooltip for svg icons since it appears that it cannot [getFeaturesAtPixel without height and width properties on the svg](https://stackoverflow.com/questions/69212440/openlayer-add-a-feature-to-layer-but-can-not-get-the-feature-by-click-event-how).